### ISSUE TITLE
Release Builds Location?

### ISSUE THREAD

**Issue by zorrion** [open-quantum-safe/liboqs#1918]
Is it possible for the project to setup a CI/CD and make a release build location for the C library so that everyone doesn't have to rebuild it constantly? 

This would make it infinitely easier to test the demos etc.

If there is a release location. Can you update this issue with the location?

**Comment by SWilson4** [comment]
Hi @zorrion, thanks for the suggestion. I would guess you're thinking of having binaries published with each release, for example in the "Assets" section [here](https://github.com/open-quantum-safe/liboqs/releases/tag/0.10.1).

I don't think it would be too difficult to publish builds for a couple of popular configurations (e.g., x86_64 portable build / standards-track algorithms enabled). I imagine we'd have to limit it to static library builds only. Any reason not to do this @baentsch @dstebila @praveksharma?

We do have a backlog of "nice-to-have" feature requests, so if this is an addition that would save you time/effort, please consider using some of that time/effort to implement it and open a PR.

**Comment by baentsch** [comment]
> Any reason not to do this 

Yes. Support and maintenance obligations. I had this for some time in oqsprovider but removed it as per https://github.com/open-quantum-safe/oqs-provider/issues/94#issuecomment-2131312561.

The rationale given

> This would make it infinitely easier to test the demos etc.

is not quite clear to me: @zorrion we have fully-building and readily built docker images for the demos. I'd have thought that's maximum ease of use (no need to build anything, not even link the library), no?

**Comment by SWilson4** [comment]
Closing as not planned—at least for now.
